sdk/communication/AzureCommunicationChat/Tests/ChatThreadClientTests.swift

```diff
@@ -217,6 +217,51 @@ class ChatThreadClientTests: XCTestCase {
         }
     }
 
+    func test_SendMessage_SendsMetadata() {
```

Comment:
Since metadata is `[String:String?]` we might want a test case to test the nil case. 
`["testMetadata": nil]`
What would we expect the result to be?

Reply:
Good point, added nil to test case. If you do send nil value expectation is metadata you get back would contain the property ie `"testMetadata": nil`